calico/felix/felix.py

```diff
@@ -242,6 +243,13 @@ def dump_top_level_actors(log):
             _log.warning("Unable to install diag dump handler")
             pass
 
+        # Count cluster existence
+        if config.USAGE_COUNT:
+            calico_version = pkg_resources.require("calico")[0].version
```

Comment:
Is the version a string; I thought it was a class of some sort.  You might want to stringify it to avoid any issues with passing it as a query param.